src/scripts.js

```diff
@@ -41,8 +36,9 @@ const fetchAllData = (userID) => {
 }
 
 const validateCustomerLogin = () => {
-  const customerLoginNumber = parseInt(username.value.substring(8))
-  if (customerLoginNumber > 0 && customerLoginNumber < 51 && password.value === "overlook2021") {
+  const loginName = username.value;
+  const customerLoginNumber = parseInt(loginName.substring(8))
+  if (loginName.startsWith("customer") && customerLoginNumber > 0 && customerLoginNumber < 51 && password.value === "overlook2021") {
```

Comment:
Perhaps a not-so-edge case where a user COUD put in anything with the same characters as 'customer' with a two-digit number at the end and still get into the site. This is not possible now